### ISSUE TITLE
events: add default blurb if eventbrite registration is available

### ISSUE THREAD

**Issue by hotzeplotz** [lsecities/lsecities-wp-theme#165]
before an event, if an eventbrite registration URI is defined for an event, we should display in the event's main text area the following blurb:

> Please register for a ticket [linked to the Eventbrite page]
> This event is free and open to all. Entry is on a first-come, first-served basis. Not everyone who registers attends our events, so to ensure a full house, we allow more registrations than there are places. Our events are very well attended, so please make sure you arrive early to avoid disappointment. We cannot guarantee entry.

Obviously this paragraph should disappear once the event has happened.

Moreover, since we may in the future run events for which registration is **not** free or open to all (e.g. invitation-only workshops and seminars), we also need a tick box (**Invitation-only event?**), set to no by default. When set to yes, no ticket blurb is displayed.

For public events with no EventBrite URI, a blurb informing that no ticket is required should be displayed instead.

Finally, to allow editors to easily manage events for which custom ticket information is needed, no ticket blurb is displayed even for events open to all and where an EventBrite URI is defined, *if any text is added to the **Event-specific ticket notice** field*. 

Basically, the logic is:

```
if is_future_event
  if ticket_notice is not empty
    # display whatever is set in ticket_notice
  else # (ticket_notice is empty)
    if not private_event
      if eventbrite_uri
        # display ticketed event blurb with link to eventbrite page
      else # (no eventbrite uri)
        # display open, unticketed event blurb
```

tasks:

* [x] add `Invitation only event?` radio (yes/no) field to Events pod
* [x] define copy for free entry
* [x] output blurb pre-event in template
